.github/workflows/build.yaml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    types: [ opened, synchronize ]
+
+env:
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+
+jobs:
+  build:
+    name: Build ${{ matrix.configuration }}
+
+    strategy:
+      matrix:
+        os: [ windows-latest ]
+        configuration: [ Debug, Release ]
+
+    runs-on: ${{ matrix.os }}
```

Comment:
Since there's only one value for `os` we technically don't need to use a matrix here, though this does make it easier to add macos-latest or ubuntu-latest later.